project/src/main/resources/application.properties

```diff
@@ -7,4 +7,4 @@ debug=false
 server.port = 8080
 
 #Connection to mongo DB
-spring.data.mongodb.uri=mongodb+srv://se452admin:<password>@se452cluster-opf6e.mongodb.net/test?retryWrites=true&w=majority
+spring.data.mongodb.uri=mongodb+srv://se452admin:<password>@se452cluster-opf6e.mongodb.net/se452?retryWrites=true&w=majority
```

Comment:
@LilyFisher I see you added some sample data into the se452 db. I got tripped up for a long time wondering why I couldn't read the sample data I had added. Eventually I figured out I had the app configured to look at the wrong database. Here's the switch just in case you were running into similar issues.